src/appendices/string-method-examples/indexOf-examples.rst

```diff
@@ -0,0 +1,43 @@
+.. _string-indexof-examples:
+
+**indexOf** Examples
+====================
+
+The general syntax for this method is:
+
+.. sourcecode:: js
+
+   stringName.indexOf(character);
```

Comment:
As commented on the string methods page, ``indexOf`` can search for multi-character strings as well.